taipy/core/_repository/_filesystem_repository.py

```diff
@@ -123,17 +123,11 @@ def _export(self, entity_id: str, folder_path: Union[str, pathlib.Path]):
         else:
             folder = folder_path
 
-        if folder.resolve() == self._storage_folder.resolve():
-            raise InvalidExportPath("The export folder must not be the storage folder.")
```

Comment:
@toan-quach asked:
> why are we removing this code?

This is moved to the `taipy.import()` since we only need to check it once.